crates/cxx-qt-gen/src/parser/cxxqtdata.rs

```diff
@@ -8,35 +8,27 @@ use crate::syntax::{
     attribute::{attribute_find_path, attribute_tokens_to_ident},
     path::path_to_single_ident,
 };
-use std::collections::HashMap;
+use std::collections::BTreeMap;
 use syn::{spanned::Spanned, Error, Ident, Item, ItemEnum, ItemImpl, Result, Type, TypePath};
 
 #[derive(Default)]
 pub struct ParsedCxxQtData {
     /// Map of the QObjects defined in the module that will be used for code generation
-    pub qobjects: HashMap<Ident, ParsedQObject>,
+    pub qobjects: BTreeMap<Ident, ParsedQObject>,
```

Comment:
There's no functional reason this changed, it's only for performance, right?

Reply:
Ah, that's a good point :thinking: 
Can you add a comment explaining this 
```suggestion
    // We have to use a BTreeMap here, instead of a HashMap, to keep the order of QObjects stable.
    // Otherwise, the output order would be different, depending on the environment, which makes it hard to test/debug.
    pub qobjects: BTreeMap<Ident, ParsedQObject>,
```

Reply:
Done